build-system/test-configs/config.js

```diff
@@ -254,6 +254,7 @@ const htmlFixtureGlobs = [
   '!examples/amp-script/vue-todomvc.amp.html',
   '!examples/amp-skimlinks.html',
   '!examples/amp-smartlinks.html',
+  '!examples/amp-story-subscription/amp-story-subscription.html',
```

Comment:
can we add a TODO to remove this when validator is updated?

Reply:
Done.